rasa/shared/core/domain.py

```diff
@@ -1789,38 +1798,116 @@ def validate(mapping: Dict[Text, Any], form_name: Text, slot_name: Text) -> None
         if not isinstance(mapping, dict):
             raise InvalidDomain(
                 f"Please make sure that the slot mappings for slot '{slot_name}' in "
-                f"your form '{form_name}' are valid dictionaries. Please see "
-                f"{DOCS_URL_FORMS} for more information."
+                f"your domain are valid dictionaries. Please see "
+                f"the documentation for more information."
```

Comment:
Maybe we want to refer to the docs URL for domains here? I assume that's where this info will live down the line (though could be subject to change at this stage I suppose)

Reply:
I actually think the information about slot mappings will be moved to the Domain/Slots page so will update accordingly. I've already made a note to revisit all these Docs links once the docs draft is ready.